### PR TITLE
All timeouts added

### PR DESCRIPTION
**Pull request by Sahiltheram**
Timeouts added.

### CODE REVIEW

**Review by MatthewSpofford — DISMISSED** [review]
It may be better to increase these timeouts from 3 seconds to 5-10 seconds for now, especially since we don't know how quickly any of these commands will run during testing. All of these timeouts should also be defined in `Constants.java` just to make it easier to change/reference later.

File: src/main/java/frc/robot/commands/arm/ManualMoveArm.java [hunk 1]
```diff
@@ -32,7 +36,7 @@ public void end(boolean Interrupted) {
 
     @Override
     public boolean isFinished() {
-        return false;
+        return Timer.getFPGATimestamp() - startTime > Constants.MANUAL_TIMEOUT;
```
Comment (bengold22) [inline src/main/java/frc/robot/commands/arm/ManualMoveArm.java]: Don't change this from returning false

File: src/main/java/frc/robot/commands/arm/VoltageMoveArm.java [hunk 2]
```diff
@@ -41,7 +43,7 @@ public void end(boolean Interrupted) {
 
     @Override
     public boolean isFinished() {
-        return Math.abs(angle - arm.getEncoderValue()) < Constants.ARM_MOVE_PID_THRESHOLD;
+        return Math.abs(angle - arm.getEncoderValue()) < Constants.ARM_MOVE_PID_THRESHOLD || Timer.getFPGATimestamp() - startTime > Constants.TIMEOUT;
```
Comment (bengold22) [inline src/main/java/frc/robot/commands/arm/VoltageMoveArm.java]: Couple of thoughts:
* Put some parentheses around these conditions just to make the logic clearer.
* Create a unique constant, and set it to be 5 seconds for now

File: src/main/java/frc/robot/commands/drive/Forward.java [hunk 3]
```diff
@@ -38,6 +41,6 @@ public void execute(){
 
     @Override
     public boolean isFinished() {   
-        return drivetrain.getRelEnc(Constants.DRIVE_BACK_LEFT_D)-InitEncValue>Constants.WHEEL_RADIUS*2*Math.PI;
+        return drivetrain.getRelEnc(Constants.DRIVE_BACK_LEFT_D)-InitEncValue>Constants.WHEEL_RADIUS*2*Math.PI || Timer.getFPGATimestamp() - startTime > Constants.TIMEOUT;
```
Comment (bengold22) [inline src/main/java/frc/robot/commands/drive/Forward.java]: Make a unique timeout constant for this and set it to 5 seconds for now

File: src/main/java/frc/robot/commands/extender/ExtendToPosition.java [hunk 4]
```diff
@@ -40,6 +43,6 @@ public void execute(){
 
     @Override
     public boolean isFinished() {
-        return (Math.abs(extender.getEncoder()-position) < Constants.EXTENDER_ERROR_THRESHOLD);
+        return (Math.abs(extender.getEncoder()-position) < Constants.EXTENDER_ERROR_THRESHOLD) || Timer.getFPGATimestamp() - startTime > Constants.TIMEOUT;
```
Comment (bengold22) [inline src/main/java/frc/robot/commands/extender/ExtendToPosition.java]: * Put some parentheses around the logical states
* Create a unique constant for this 
* Set the time to 5 seconds for now

File: src/main/java/frc/robot/commands/extender/ManualExtender.java [hunk 5]
```diff
@@ -36,6 +39,6 @@ public void execute(){
 
     @Override
     public boolean isFinished() {
-        return false;
+        return Timer.getFPGATimestamp() - startTime > Constants.MANUAL_TIMEOUT;
```
Comment (bengold22) [inline src/main/java/frc/robot/commands/extender/ManualExtender.java]: Set this back to return false

File: src/main/java/frc/robot/commands/extender/ManualMoveExtender.java [hunk 6]
```diff
@@ -25,7 +36,7 @@ public void execute() {
 
     @Override
     public boolean isFinished() {
-        return false;
+        return Timer.getFPGATimestamp() - startTime > Constants.MANUAL_TIMEOUT;
```
Comment (bengold22) [inline src/main/java/frc/robot/commands/extender/ManualMoveExtender.java]: Return false here

File: src/main/java/frc/robot/commands/extender/ResetExtenderEncoder.java [hunk 7]
```diff
@@ -37,10 +42,6 @@ public void end(boolean interrupted) {
 
     @Override
     public boolean isFinished() {
-        return limitReached;
+        return limitReached || Timer.getFPGATimestamp() - startTime > Constants.TIMEOUT;
```
Comment (bengold22) [inline src/main/java/frc/robot/commands/extender/ResetExtenderEncoder.java]: Make a unique constant for the timeout and make it 5 seconds for now

File: src/main/java/frc/robot/commands/gripper/ManualMoveGripper.java [hunk 8]
```diff
@@ -28,12 +31,12 @@ public void execute() {
...
     @Override
     public void initialize() {
-
+        startTime = Timer.getFPGATimestamp();
     }
 
     @Override
     public boolean isFinished() {
-        return false;
+        return Timer.getFPGATimestamp() - startTime > Constants.MANUAL_TIMEOUT;
```
Comment (bengold22) [inline src/main/java/frc/robot/commands/gripper/ManualMoveGripper.java]: Make this return false

**Review by bengold22 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**